test/conftest.py

```diff
@@ -21,6 +21,8 @@ def pytest_collection_modifyitems(items) -> None:
         relative_path = item_path.relative_to(top_test_dir)
         for part in relative_path.parts:
             item.add_marker(part)
+            if part in ('unit', 'pandas'):
+                item.add_marker('skipolddriver')
```

Comment:
skipping old test driver if `unit` or `pandas` test.

In the next chance of old driver version change, `unit` still needs to be excluded but `pandas` may need to be added.

@sfc-gh-mkeller , which on do you like? this or adding `skipolddriver` to `pandas` modules individually?

Reply:
I'd generally like not adding them one-by-one, but in that case we'd have to keep a good lists of which tags are not run on the old drivers.
However, I'm open to add them manually if you think that being more explicit would be beneficial.

Reply:
I'm ok with either case. It is more or less up to you which is good for your maintenance.